pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -2,36 +2,36 @@
   "dependencies": {
     "Microsoft.Net.Native.Compiler": "1.5.0",
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
-    "Microsoft.NETCore.Targets": "1.0.2",
-    "Microsoft.Win32.Primitives": "4.0.1",
-    "System.ComponentModel.EventBasedAsync": "4.0.11",
-    "System.Data.Common": "4.1.0",
-    "System.Diagnostics.Contracts": "4.0.1",
-    "System.Diagnostics.StackTrace": "4.0.2",
-    "System.IO.IsolatedStorage": "4.0.1",
-    "System.Net.Http.Rtc": "4.0.1",
-    "System.Net.NameResolution": "4.0.0",
-    "System.Net.Requests": "4.0.11",
-    "System.Net.Sockets": "4.1.0",
-    "System.Net.WebHeaderCollection": "4.0.1",
-    "System.Net.WebSockets": "4.0.0",
-    "System.Net.WebSockets.Client": "4.0.0",
-    "System.Numerics.Vectors.WindowsRuntime": "4.0.1",
-    "System.Reflection.Context": "4.0.1",
-    "System.Runtime.InteropServices.WindowsRuntime": "4.0.1",
-    "System.Runtime.Serialization.Json": "4.0.2",
-    "System.Runtime.Serialization.Primitives": "4.1.1",
-    "System.Runtime.Serialization.Xml": "4.1.1",
-    "System.Runtime.WindowsRuntime": "4.0.11",
-    "System.Runtime.WindowsRuntime.UI.Xaml": "4.0.1",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24708-01",
```

Comment:
When VersionTools is involved, stable dependencies stay (by default), but core-setup has a custom implementation.

I'll configure it so these files aren't updated.